src/Compilers/VisualBasic/Portable/Syntax/VisualBasicSyntaxTree.ParsedSyntaxTree.vb

```diff
@@ -123,13 +123,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return New SimpleSyntaxReference(Me, node)
             End Function
 
-            ''' <summary>
-            ''' Returns a <see cref="String" /> that represents the source code of this parsed tree.
-            ''' </summary>
-            Public Overrides Function ToString() As String
-                Return Me.GetText().ToString()
-            End Function
-
```

Comment:
@AnthonyDGreen Does VB call the most derived override known statically?  Or does it behave like C#?  tnx!